vocab/properties.json

```diff
@@ -130,38 +168,51 @@
         "label": "Acknowledgement",
         "labelForReverseLink": "Is acknowledgement of",
         "name": "acknowledgement",
+        "nameForReverseLink": "isAcknowledgementOf",
         "sameAs": null,
         "schemas": [
             "core/v0/miscellaneous/funding",
             "core/v3/miscellaneous/funding",
             "core/v4/miscellaneous/funding"
-        ]
+        ],
+        "formating": {
+            "formattedText": true,
+            "multilineText": true
+        }
     },
     "https://openminds.ebrains.eu/vocab/addExistingTerminology": {
```

Comment:
this property needs to be changed. this includes the instruction which should be avoided